internal/internal_event_handlers.go

```diff
@@ -1159,10 +1159,10 @@ func (weh *workflowExecutionEventHandlerImpl) ProcessLocalActivityResult(lar *lo
 	if lar.err != nil {
 		errReason, errDetails := getErrorDetails(lar.err, weh.GetDataConverter())
 		lamd.ErrReason = errReason
-		lamd.ErrJSON = string(errDetails)
+		lamd.ErrJSON = string(errDetails) // TODO: this is not json, nor is it necessarily a valid string
```

Comment:
clarification, not *validated* as being json or even a string? or *never* json?

Reply:
it's arbitrary user-encoded data (because it uses their dataconverter).

json is the default, but since it's arbitrary IMO that means "never json".